app/src/main/java/com/peacecorps/pcsa/circle_of_trust/Trustees.java

```diff
@@ -305,5 +323,43 @@ private boolean noDuplicateNumber() {
         }
         return noDuplicate;
     }
+
+    /**
+     * Set focus to the next EditText after picking contact
+     */
+    public void setFocusOnNextView() {
```

Comment:
Sorry if I confused you What I meant to tell is that the phoneInput is the selected button. The previous developer has done the same thing when creating the phoneInput, you can use that without doing the same thing again. Please go through the current implementation and try to understand what is done there. Then you will be able to implement this more easily.